content/en/guides/models/app/features/panels/line-plot/_index.md

```diff
@@ -168,4 +169,52 @@ From a line plot, you can quickly create a [run metrics notification]({{< relref
 1. Hover over the panel, then click the bell icon.
 1. Configure the automation using the basic or advanced configuration controls. For example, apply a run filter to limit the scope of the automation, or configure an absolute threshold.
 
-Learn more about [Automations]({{< relref "/guides/core/automations" >}}).
\ No newline at end of file
+Learn more about [Automations]({{< relref "/guides/core/automations" >}}).
+
+## Visualize CoreWeave infrastructure alerts
+
+{{< alert>}}
+This feature is in Preview and only available when training on a CoreWeave cluster. Contact your W&B representative for access.
+{{< /alert >}}
+
+Observe infrastructure alerts such as GPU failures, thermal violations, and more during machine learning experiments you log to W&B. During a [W&B run]({{< relref "/guides/models/track/runs/_index" >}}), [CoreWeave Mission Control](https://www.coreweave.com/mission-control) monitors your compute infrastructure. If an error occurs, CoreWeave sends that information to W&B. W&B populates infrastructure information onto your run’s plots in your project’s workspace. CoreWeave attempts to automatically resolve some issues, and W&B surfaces that information in the run’s page.
```

Comment:
In the HTML preview, this paragraph looks a bit big and blocky. I wonder if it would be helpful to move the alert to after "experiments you log to W&B." or after "monitors your compute infrastructure." to break it up.